staging/src/slime.io/slime/modules/lazyload/controllers/update_resources.go

```diff
@@ -0,0 +1,185 @@
+package controllers
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"os"
+	config "slime.io/slime/framework/apis/config/v1alpha1"
+	"slime.io/slime/framework/bootstrap"
+	"slime.io/slime/modules/lazyload/api/v1alpha1"
+	"sort"
+	"strings"
+)
+
+var (
+	slimeBootGvr = schema.GroupVersionResource{
+		Group:    "config.netease.com",
+		Version:  "v1alpha1",
+		Resource: "slimeboots",
+	}
+	defaultPort      = 80
+	defaultProbePort = 18181
+	defaultReplicas  = 1
+	defaultIstioNs = "istio-system"
+	defaultSlimeNs = "mesh-operator"
+)
+
+type SlimeBoot struct {
+	metav1.TypeMeta   `json:",inline,omitempty" yaml:",inline,omitempty"`
+	metav1.ObjectMeta `json:"metadata,omitempty" yaml:"metadata,omitempty"`
+	Spec              *SlimeBootSpec `json:"spec" yaml:"spec"`
+}
+
+type SlimeBootSpec struct {
```

Comment:
i still can not understand why we should claim these types here

Reply:
add todo: change these structs to framework